src/types.ts

```diff
@@ -1726,6 +2183,26 @@ export interface KAPLAYCtx<
     ): KEventController;
     /**
      * Register an event that runs when the gamepad axis exists.
+     * 
+     * @example
+     * ```js
+     * // player move
+     * let player = add([
+     *     pos(center()),
+     *     anchor(`center`),
+     *     sprite(`bean`),
+     *     area(),
+     *     body(),
+     *     {
```

Comment:
Example doesn't need all this components, with pos and sprite is already understable